packages/client/test/usage.spec.ts

```diff
@@ -0,0 +1,95 @@
+import nock from 'nock';
+import './setup';
+import { Usage } from '../src';
+import Sinon from 'sinon';
+import * as config from '../src/loadConfiguration';
+import { ApiKey } from './setup';
+
+describe('Usage', () => {
+  beforeAll(() => {
+    nock.disableNetConnect();
+  });
+
+  const addDays = (date: Date, days: number) => {
+    const newDate = new Date(date);
+    newDate.setDate(date.getDate() + days);
+    return newDate;
+  };
+
+  const usageReport = {
+    events: {
+      usage: 0,
+      limit: 10000,
+    },
+    cycleStart: new Date().toISOString(),
+    cycleEnd: addDays(new Date(), 30).toISOString(),
+  };
+
+  beforeEach(() => {
+    Sinon.restore();
+    Sinon.stub(config, 'default').returns({
+      baseURL: config.DefaultURL,
+      apiURL: 'http://localhost:3000',
+      apiKey: ApiKey,
+    });
+  });
+
+  describe('get', () => {
+    it('issues an expected API call without an org identifier', () => {
+      nock('http://localhost:3000')
+        .get('/v1/orgs/usage')
+        .query({})
+        .matchHeader('authorization', `Bearer ${ApiKey}`)
+        .reply(200, usageReport);
+      return expect(Usage.get()).resolves.toEqual(usageReport);
+    });
+
+    it('issues an expected API call when an org identifier is provided', () => {
+      const orgId = 1;
+      nock('http://localhost:3000')
+        .get('/v1/orgs/usage')
```

Comment:
Query to this data will be restricted by org id so I’d expect that to be a path parameter?